tests/MongoDB.Bson.Tests/Serialization/BsonClassMapSerializerISupportInitializeTests.cs

```diff
@@ -13,6 +13,7 @@
 * limitations under the License.
 */
 
+#if !NETCOREAPP1_0
```

Comment:
`System.ComponentModel.ISupportInitialize` is not available in netstandard1.5 :)

Reply:
Fwiw, it's also not available in netcoreapp1.1, so if we decide to change the framework target for the test project, this ifdef will need to be updated